frontend/src/constants/index.js

```diff
@@ -151,3 +151,11 @@ export const defaultPhrases = [
   "Мы, к сожалению, не можем ответить на поставленный Вами вопрос ",
   "Поздравляем! С Вами свяжутся по электронной почте ",
 ];
+
+export const possibleLanguages = ["JAVA", "JAVA_SCRIPT", "PYTHON"];
```

Comment:
```suggestion
export const possibleLanguages = Object.keys(mapIdToButtonName);
```
чтоб не повторять, можно еще переименовать переменную

Reply:
Согласен, поправил eac2b0d20eadc056ea4be24f93a62e600716c45b